apps/ucp-ui/package.json

```diff
@@ -22,6 +22,7 @@
     "@mui/lab": "^5.0.0-alpha.173",
     "@mui/material": "^5.16.6",
     "@reduxjs/toolkit": "^2.2.7",
+    "@repo/shared-utils": "^0.0.1",
```

Comment:
This isn't actually used in ucp-ui right?

Reply:
It is. Check `App.tsx`

Reply:
It is now.